### ISSUE TITLE
Ember build error after installing ember-fontawesome

### ISSUE THREAD

**Issue by amiarSlimane** [FortAwesome/ember-fontawesome#199]
**Describe the bug**
i am getting the following error when running "ember serve" after i have installed ember-fontawesome

```

Build Error (broccoli-persistent-filter:Babel > [Babel: @embroider/macros]) in @embroider/macros/runtime.js

/home/amiar/makane/promotion/generate-data/@embroider/macros/runtime.js: @babel/template placeholder "APP": Expected string substitution


Stack Trace and Error Report: /tmp/error.dump.59e033821933915a2d87abdab151ddb1.log
```

**Reproducible test case**

`ember install @fortawesome/ember-fontawesome`
`ember serve `

 

**Desktop (please complete the following information):**
 
 - Version  
       "ember-source": "~3.26.1",
       "ember-cli": "~3.26.1",
 

## Stack Trace and Error Report 

```

=================================================================================

ENV Summary:

  TIME: Mon Feb 07 2022 10:46:31 GMT+0100 (Central European Standard Time)
  TITLE: ember
  ARGV:
  - /usr/bin/node
  - /usr/bin/ember
  - s
  EXEC_PATH: /usr/bin/node
  TMPDIR: /tmp
  SHELL: /bin/bash
  PATH:
  - /home/amiar/.locale/bin
  - /home/amiar/.yarn/bin
  - /home/amiar/.config/yarn/global/node_modules/.bin
  - /home/amiar/.local/bin
  - /usr/local/sbin
  - /usr/local/bin
  - /usr/sbin
  - /usr/bin
  - /sbin
  - /bin
  - /usr/games
  - /usr/local/games
  - /snap/bin
  - /snap/bin
  - /home/amiar/goprojects/bin
  - /home/amiar/go/bin
  - /bin
  - /usr/lib/lightdm/lightdm
  - /usr/local/sbin
  - /usr/local/bin
  - /usr/sbin
  - /usr/bin
  - /sbin
  - /bin
  - /usr/games
  - /usr/local/games
  - /snap/bin
  - /home/amiar/Android/Sdk/platform-tools
  - /usr/local/go/bin
  - /home/amiar/.local/bin
  - /home/amiar/go/bin
  PLATFORM: linux x64
  FREEMEM: 7056314368
  TOTALMEM: 25084067840
  UPTIME: 10357.36
  LOADAVG: 2.16,2.19,2.33
  CPUS:
  - Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz - 1900
  - Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz - 1900
  - Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz - 1900
  - Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz - 2848
  - Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz - 1900
  - Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz - 1900
  - Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz - 1900
  - Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz - 1900
  ENDIANNESS: LE
  VERSIONS:
  - ares: 1.18.1
  - brotli: 1.0.9
  - cldr: 40.0
  - icu: 70.1
  - llhttp: 2.1.4
  - modules: 83
  - napi: 8
  - nghttp2: 1.42.0
  - node: 14.19.0
  - openssl: 1.1.1m
  - tz: 2021a3
  - unicode: 14.0
  - uv: 1.42.0
  - v8: 8.4.371.23-node.85
  - zlib: 1.2.11

ERROR Summary:

  - broccoliBuilderErrorStack: Error: @babel/template placeholder "APP": Expected string substitution
    at applyReplacement (/home/amiar/makane/promotion/generate-data/node_modules/@babel/template/lib/populate.js:74:13)
    at /home/amiar/makane/promotion/generate-data/node_modules/@babel/template/lib/populate.js:44:7
    at Array.forEach (<anonymous>)
    at populatePlaceholders (/home/amiar/makane/promotion/generate-data/node_modules/@babel/template/lib/populate.js:42:43)
    at /home/amiar/makane/promotion/generate-data/node_modules/@babel/template/lib/string.js:20:51
    at /home/amiar/makane/promotion/generate-data/node_modules/@babel/template/lib/builder.js:75:14
    at Object.buildLiterals (/home/amiar/makane/promotion/generate-data/node_modules/@embroider/macros/src/babel/evaluate-json.js:377:73)
    at Object.inlineRuntimeConfig (/home/amiar/makane/promotion/generate-data/node_modules/@embroider/macros/src/babel/get-config.js:107:55)
    at PluginPass.enter (/home/amiar/makane/promotion/generate-data/node_modules/@embroider/macros/src/babel/macros-babel-plugin.js:77:38)
    at newFn (/home/amiar/makane/promotion/generate-data/node_modules/@babel/traverse/lib/visitors.js:177:21)
    =============
    at Object.buildLiterals (/home/amiar/makane/promotion/generate-data/node_modules/@embroider/macros/src/babel/evaluate-json.js:377:34)
    at Object.inlineRuntimeConfig (/home/amiar/makane/promotion/generate-data/node_modules/@embroider/macros/src/babel/get-config.js:107:55)
    at PluginPass.enter (/home/amiar/makane/promotion/generate-data/node_modules/@embroider/macros/src/babel/macros-babel-plugin.js:77:38)
    at newFn (/home/amiar/makane/promotion/generate-data/node_modules/@babel/traverse/lib/visitors.js:177:21)
    at NodePath._call (/home/amiar/makane/promotion/generate-data/node_modules/@babel/traverse/lib/path/context.js:53:20)
    at NodePath.call (/home/amiar/makane/promotion/generate-data/node_modules/@babel/traverse/lib/path/context.js:40:17)
    at NodePath.visit (/home/amiar/makane/promotion/generate-data/node_modules/@babel/traverse/lib/path/context.js:100:31)
    at TraversalContext.visitQueue (/home/amiar/makane/promotion/generate-data/node_modules/@babel/traverse/lib/context.js:103:16)
  - code: [undefined]
  - codeFrame: /home/amiar/makane/promotion/generate-data/@embroider/macros/runtime.js: @babel/template placeholder "APP": Expected string substitution
  - errorMessage: @embroider/macros/runtime.js: /home/amiar/makane/promotion/generate-data/@embroider/macros/runtime.js: @babel/template placeholder "APP": Expected string substitution
        in /tmp/broccoli-194932lez3uh3atrAk/out-163-funnel_funnel_embroider_macros_addon
        at broccoli-persistent-filter:Babel > [Babel: @embroider/macros] (Babel: @embroider/macros)
  - errorType: Build Error
  - location:
    - column: [undefined]
    - file: @embroider/macros/runtime.js
    - line: [undefined]
    - treeDir: /tmp/broccoli-194932lez3uh3atrAk/out-163-funnel_funnel_embroider_macros_addon
  - message: @embroider/macros/runtime.js: /home/amiar/makane/promotion/generate-data/@embroider/macros/runtime.js: @babel/template placeholder "APP": Expected string substitution
        in /tmp/broccoli-194932lez3uh3atrAk/out-163-funnel_funnel_embroider_macros_addon
        at broccoli-persistent-filter:Babel > [Babel: @embroider/macros] (Babel: @embroider/macros)
  - name: Error
  - nodeAnnotation: Babel: @embroider/macros
  - nodeName: broccoli-persistent-filter:Babel > [Babel: @embroider/macros]
  - originalErrorMessage: /home/amiar/makane/promotion/generate-data/@embroider/macros/runtime.js: @babel/template placeholder "APP": Expected string substitution
  - stack: Error: @babel/template placeholder "APP": Expected string substitution
    at applyReplacement (/home/amiar/makane/promotion/generate-data/node_modules/@babel/template/lib/populate.js:74:13)
    at /home/amiar/makane/promotion/generate-data/node_modules/@babel/template/lib/populate.js:44:7
    at Array.forEach (<anonymous>)
    at populatePlaceholders (/home/amiar/makane/promotion/generate-data/node_modules/@babel/template/lib/populate.js:42:43)
    at /home/amiar/makane/promotion/generate-data/node_modules/@babel/template/lib/string.js:20:51
    at /home/amiar/makane/promotion/generate-data/node_modules/@babel/template/lib/builder.js:75:14
    at Object.buildLiterals (/home/amiar/makane/promotion/generate-data/node_modules/@embroider/macros/src/babel/evaluate-json.js:377:73)
    at Object.inlineRuntimeConfig (/home/amiar/makane/promotion/generate-data/node_modules/@embroider/macros/src/babel/get-config.js:107:55)
    at PluginPass.enter (/home/amiar/makane/promotion/generate-data/node_modules/@embroider/macros/src/babel/macros-babel-plugin.js:77:38)
    at newFn (/home/amiar/makane/promotion/generate-data/node_modules/@babel/traverse/lib/visitors.js:177:21)
    =============
    at Object.buildLiterals (/home/amiar/makane/promotion/generate-data/node_modules/@embroider/macros/src/babel/evaluate-json.js:377:34)
    at Object.inlineRuntimeConfig (/home/amiar/makane/promotion/generate-data/node_modules/@embroider/macros/src/babel/get-config.js:107:55)
    at PluginPass.enter (/home/amiar/makane/promotion/generate-data/node_modules/@embroider/macros/src/babel/macros-babel-plugin.js:77:38)
    at newFn (/home/amiar/makane/promotion/generate-data/node_modules/@babel/traverse/lib/visitors.js:177:21)
    at NodePath._call (/home/amiar/makane/promotion/generate-data/node_modules/@babel/traverse/lib/path/context.js:53:20)
    at NodePath.call (/home/amiar/makane/promotion/generate-data/node_modules/@babel/traverse/lib/path/context.js:40:17)
    at NodePath.visit (/home/amiar/makane/promotion/generate-data/node_modules/@babel/traverse/lib/path/context.js:100:31)
    at TraversalContext.visitQueue (/home/amiar/makane/promotion/generate-data/node_modules/@babel/traverse/lib/context.js:103:16)

=================================================================================


```

**Comment by kariavka** [comment]
The same issue for me

**Comment by st-h** [comment]
I have been able to work around this by pinning `ember-get-config` to version `0.3.0` (not a long term solution).
Afaik, this should be fixed in version 1.0.0 (which this addon already has as a dependency). Likely there is another addon in your project that pulls in a different version of `ember-get-config` which needs to be updated.

**Comment by jrjohnson** [comment]
Duplicate of #194 which describes some solutions along with what @st-h has said here.
